### PR TITLE
docs: tutorial for submitting bcs txn with TS SDK

### PR DESCRIPTION
**Pull request by jjleng**
### Description
Tutorial to show how to submit BCS transactions with TS SDK

### Test Plan
Previewed locally

---
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/1527)

### CODE REVIEW

File: developer-docs-site/docs/sdks/aptos-sdk-overview.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+Vector items are self-explanatory. However, special serialization method is used to save space and avoid data overflow. Details are described below:
+
+- All account address are hex-coded.
+- `args` in ScriptFunction is hex-coded.
+- U64 and U128 numbers are serialized as string literals to avoid data truncation.
+- U8 is serialized as a number (not a string).
+- `code` in Script and ModuleBundle are hex-coded.
+
+See how Typescript SDK does vector validation [here](https://github.com/aptos-labs/aptos-core/pull/1461).
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/sdks/aptos-sdk-overview.md:133]: Just alerting here.

File: developer-docs-site/docs/sdks/aptos-sdk-overview.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+The transaction builder contains the Typescript types for constructing the transaction payloads. The Typescript SDK supports three kinds of transaction payloads:
+
+1. ScriptFunction
+2. Script
+3. ModuleBundle
+
+### Aptos Client
+
+The Aptos Client is the main component of Typescript SDK. It exposes the methods for retrieving the account resources, transactions, and modules.
```
Comment (jjleng) [inline developer-docs-site/docs/sdks/aptos-sdk-overview.md:82]: account resources, transactions, modules and events

File: developer-docs-site/docs/sdks/aptos-sdk-overview.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+:::note
+
+The second method, i.e., in BCS format, is the recommended way for submitting transactions to the Aptos Blockchain.
+
+:::
+
+### Token Client
+
+The Token Clientp provides methods for the creation, token minting, and token querying of the NFT collection.
```
[the remark this answers is not in the document]
Comment (jjleng) [reply developer-docs-site/docs/sdks/aptos-sdk-overview.md:97]: maybe `methods for the creation, and querying of the NFT collections and tokens`

**Comment by davidiw** [comment]
not sure what's going on here

**Pull request closed without merging**